src/Uuid/Uuid.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Uuid;
+
+/**
+ * @internal
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class Uuid
+{
+    const UUID_VARIANT_NCS = 0;
+    const UUID_VARIANT_DCE = 1;
+    const UUID_VARIANT_MICROSOFT = 2;
+    const UUID_VARIANT_OTHER = 3;
+    const UUID_TYPE_DEFAULT = 0;
+    const UUID_TYPE_TIME = 1;
+    const UUID_TYPE_DCE = 4;
+    const UUID_TYPE_NAME = 1;
+    const UUID_TYPE_RANDOM = 4;
+    const UUID_TYPE_NULL = -1;
+    const UUID_TYPE_INVALID = -42;
+
+    public static function uuid_create($uuid_type = UUID_TYPE_DEFAULT)
+    {
+        if (!\is_int($uuid_type)) {
+            trigger_error(sprintf("uuid_create() expects parameter 1 to be int, %s given", gettype($uuid_type)), E_USER_WARNING);
+
+            return null;
+        }
+
+        switch ($uuid_type) {
+            case UUID_TYPE_NAME:
```

Comment:
should be `self::UUID_TYPE_NAME` or move constantes outside the class

Reply:
It works because the polyfill was already loaded.
But indeed, if one use the class directly (this is forbidden by `@internal` tag) it will crash.

But you are right, I will update this